packages/hardhat/contracts/minters/ERC721Minter.sol

```diff
@@ -55,6 +65,14 @@ abstract contract ERC721Minter is PaymentSplitterUpgradeable, AccessControlEnume
         address(this).functionCall(extraInitCallData_);
     }
 
+    function setServiceFeeBasisPoints(uint256 serviceFeeBasisPoints_) external onlyRole(DEPLOYER_ROLE) {
```

Comment:
what were your considerations for not emitting events upon these param changes?

Reply:
@eladmallel no consideration. I'll add events 👍